worker-test/src/main/java/com/github/workerframework/testworker/TestWorkerFactory.java

```diff
@@ -31,13 +31,16 @@ final class TestWorkerFactory implements WorkerFactory
 {
     private final TestWorkerConfiguration config;
 
+    private final ConfigurationSource configSource;
```

Comment:
Do we need these changes now?

Reply:
Kept this in because it is needed to run the integration test